README.md

```diff
@@ -43,6 +43,11 @@ rules.
 
 ### Creating a Release with Git Flow
 
+This project uses git flow for releases of changes to site framework source
+changes. Prior to making a release, we should also ask TNC whether there are
```

Comment:
`This project uses git flow for releases of changes to site framework source changes` reads a little off. Is the last `changes` supposed to `code`?

Reply:
Oops, yeah -- good catch. Will make a fix to update before merging.